tests/main/interfaces-fuse_support/task.yaml

```diff
@@ -57,8 +57,13 @@ execute: |
     test-snapd-fuse-consumer.create $MOUNT_POINT
     # use "[f]oo" to search for "foo" in ps output without having to filter yourself out
     PID=$(ps aux | awk '/[t]est-snapd.*create/{print $2}')
+    match=0
+    # there may be more than one PID that matched, at least one should be the right match
```

Comment:
Why can there be more than one?

Reply:
From what I see in the library and strace output, the hello.py helper forks at least 3 times. Come to think of it, I have no clue why we're using the forking variant. Tomorrow I'll look into running the fuse helper in foreground.